File: tony-java-monolithic-demo/tony-dto/src/main/java/com/tony/mono/dto/req/UserListQueryReq.java
package com.tony.mono.dto.req;

import com.tony.PageQueryLike;
import lombok.Getter;
import lombok.Setter;

import java.util.Set;

@Setter
@Getter
public class UserListQueryReq implements PageQueryLike {

    private Long page;

    private Long size;

    private Set<String> ascs;

    private Set<String> descs;

    private String query;
}

File: tony-monolithic-demo/tony-dto/src/main/kotlin/com/tony/dto/req/UserListQueryReq.kt
package com.tony.dto.req

import com.tony.PageQueryLike

class UserListQueryReq(
    var query: String?,
    override val page: Long = 1,
    override val size: Long = 10,
    override val ascs: MutableList<String?> = mutableListOf(),
    override val descs: MutableList<String?> = mutableListOf(),
) : PageQueryLike
